### ISSUE TITLE
v0.3.9 release

### ISSUE THREAD

**Issue by ahfarmer** [medikoo/memoize#43]
Would it be possible to get a 0.3.9 release? Anything I could do to help make that happen?

In particular, [this commit](https://github.com/medikoo/memoize/commit/453ee0d484dd899514ab11fd0fd7aeef5e42b822) would be very helpful to me.

I'm using Karma and experienced a bug with the stack trace below. The issue is fixed when I change Karma's memoize dependency from this:

    "memoizee": "^0.3.8",

to this:

    "memoizee": "git@github.com:medikoo/memoize.git",

Once there is a new memoizee release I'll be able to submit a simple PR to Karma that fixes the issue by updating the dependency from 0.3.8 to 0.3.9.

Thanks! 😸

The stack trace I mentioned above:
```
/vagrant/core/ux/node_modules/karma/node_modules/memoizee/lib/weak.js:58
				clear: d(map.clear.bind(map))
				                  ^
TypeError: Cannot read property 'bind' of undefined
    at /vagrant/core/ux/node_modules/karma/node_modules/memoizee/lib/weak.js:58:23
    at createErrorFormatter (/vagrant/core/ux/node_modules/karma/lib/reporter.js:32:30)
    at createReporters (/vagrant/core/ux/node_modules/karma/lib/reporter.js:75:24)
    at Array.invoke (/vagrant/core/ux/node_modules/karma/node_modules/di/lib/injector.js:75:15)
    at [object Object].get (/vagrant/core/ux/node_modules/karma/node_modules/di/lib/injector.js:48:43)
    at get (/vagrant/core/ux/node_modules/karma/node_modules/di/lib/injector.js:54:19)
    at /vagrant/core/ux/node_modules/karma/node_modules/di/lib/injector.js:71:14
    at Array.map (native)
    at [object Object].invoke (/vagrant/core/ux/node_modules/karma/node_modules/di/lib/injector.js:70:31)
    at createWebServer (/vagrant/core/ux/node_modules/karma/lib/web-server.js:51:19)
    at Array.invoke (/vagrant/core/ux/node_modules/karma/node_modules/di/lib/injector.js:75:15)
    at get (/vagrant/core/ux/node_modules/karma/node_modules/di/lib/injector.js:48:43)
    at /vagrant/core/ux/node_modules/karma/node_modules/di/lib/injector.js:71:14
    at Array.map (native)
    at [object Object].invoke (/vagrant/core/ux/node_modules/karma/node_modules/di/lib/injector.js:70:31)
    at [object Object].Server.start (/vagrant/core/ux/node_modules/karma/lib/server.js:92:18)
```

**Comment by edvinerikson** [comment]
I'm having the same issue with Node `v0.12.7`
https://travis-ci.org/edvinerikson/isomorphic-react-kickstarter/jobs/73673617

**Comment by medikoo** [comment]
It's published as v0.3.9, sorry for delay, actually it caught me on holiday :)

**Comment by ahfarmer** [comment]
Thank you! 😃😸🐔
